### PR TITLE
Create droplet

### LINKED ISSUE #5: Add droplet create command
`/do droplet <name>`
Only sysadmins or some other authorized party should run this command.

Create a UI element with dropdown options for `Name`, `Size`, `Region` and the `Image` used to create the droplet. 
The region, size comes from`regions, resp, err := client.Regions.List(context.TODO(), opts)`
Each region comes with an array of sizes it supports.
Image info from from `images, resp, err := client.Images.List(context.TODO(), opts)`

### PR DESCRIPTION
**Pull request by AGMETEOR**
Fixes #5

### CODE REVIEW

File: webapp/src/client.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        this.axiosInstance = axios.create({
+            baseUrl: '/plugins/com.mattermost.digitalocean',
+        });
+    }
+
+    getDOTeamRegions = async () => {
+        return this.doGet('/api/v1/get-do-regions');
+    }
+
+    getDOTeamDropletSizes = async () => {
```
Comment (phillipahereza) [inline webapp/src/client.js]: Do you know that sizes depend on the region? Some sizes are only available in specific regions. The Regions endpoint provides a list of sizes available for each region. Sample response for get regions

```
{
   "regions":[
      {
         "name":"New York 1",
         "slug":"nyc1",
         "sizes":[

         ],
         "features":[
            "virtio",
            "backups"
         ],
         "available":false
      },
      {
         "name":"Amsterdam 1",
         "slug":"ams1",
         "sizes":[

         ],
         "features":[
            "virtio",
            "backups"
         ],
         "available":false
      },
      {
         "name":"San Francisco 1",
         "slug":"sfo1",
         "sizes":[
            "s-1vcpu-1gb",
            "s-1vcpu-2gb",
            "s-1vcpu-3gb",
            "s-2vcpu-2gb",
            "s-3vcpu-1gb",
            "s-2vcpu-4gb",
            "s-4vcpu-8gb",
            "s-6vcpu-16gb",
            "s-8vcpu-32gb",
            "s-12vcpu-48gb",
            "s-16vcpu-64gb",
            "s-20vcpu-96gb",
            "s-24vcpu-128gb",
            "s-32vcpu-192gb"
         ],
         "features":[
            "virtio",
            "backups",
            "metadata"
         ],
         "available":true
      }
   ]
}
```

Comment (AGMETEOR) [reply webapp/src/client.js]: Let me work on this

Comment (AGMETEOR) [reply webapp/src/client.js]: @phillipahereza how representative is this sample response? Does that mean New York 1 has no sizes?

Comment (phillipahereza) [reply webapp/src/client.js]: It's a dummy response, I was just showing you the structure of the response

**Pull request merged**